hikari/guilds.py

```diff
@@ -889,6 +1213,36 @@ class GuildMemberBan:
     user: users.User = attr.field(repr=True)
     """The object of the user this ban targets."""
 
+    async def fetch_user(self) -> users.User:
+        """Fetch the banned user.
+
```

Comment:
Isn't this method redundant, last time i checked people can already do `await ban.user.fetch_self())

Reply:
Yes, I too think its redundant. But it was mentioned in #37, that's why I added it.

Reply:
Nah, this is redundant and can be removed. I moved the tickets from gitlab, never actually checked the content.